docs/CHANGES.rst

```diff
@@ -0,0 +1 @@
+../CHANGES.rst
```

Comment:
I can replace docs/CHANGES.rst with an actual file (not a symlink) like this:

```
Changelog
#########

.. include:: ../CHANGES.txt
```

As long as people remember when editing CHANGES.txt that it should be valid rst, I'm happy with that.